shared/src/main/scala/mlscript/helpers.scala

```diff
@@ -629,8 +641,77 @@ trait TermImpl extends StatementImpl { self: Term =>
     case Eqn(lhs, rhs) => s"${lhs.showDbg} = ${rhs.showDbg}"
     case AdtMatchWith(cond, arms) =>
       s"match ${cond.showDbg} with ${arms.map (patmat => s"${patmat.pat.showDbg} -> ${patmat.rhs.showDbg}").mkString (" | ") }"
+    case Quoted(b) => s"code\"$b\""
+    case Unquoted(b) => s"$${$b}"
     case Rft(bse, tu) => s"${bse.showDbg} ${tu.showDbg}"
   }}
+
+  def show(newDefs: Bool): Str = showIn(ShowCtx.mk(Nil, newDefs), false)
+  def showIn(ctx: ShowCtx, brackets: Bool): Str = {
```

Comment:
I regret not making `ctx: ShowCtx` an implicit parameter in `showIn` this family of functions. Maybe now is the time to make it right, since we're adding this fresh new code.